airflow/config_templates/config.yml

```diff
@@ -477,15 +477,15 @@ core:
       type: string
       default: "0o077"
       example: ~
-    dataset_manager_class:
```

Comment:
These were not deprecated. I think the plan is to add deprecation warning for whatever we are removing/renaming in 2.11. What is the plan here? Should we add them now in 2.10 branch? Or wait December to add them (then we need to keep track of all breaking changes, through significant files maybe?) @potiuk @eladkal